src/libs/ajax/GoogleStorage.ts

```diff
@@ -21,6 +22,7 @@ import {
   toolLabels
 } from 'src/pages/workspaces/workspace/analysis/tool-utils'
 
+import { Ajax } from '../ajax'
```

Comment:
Oh I really don't like this... this is a circular dependency. Ajax pulls in this module which pulls in Ajax. Metrics should likely be split out into its own module, as GoogleStorage is.